package.json

```diff
@@ -34,6 +34,8 @@
       "finance.ledgers": "1.0",
       "finance.order-transaction-summaries": "1.0",
       "finance.transactions": "3.0 4.0",
+      "invoice": "7.0",
+      "order-lines": "2.0",
```

Comment:
IIRC< these interfaces were updated recently, and there is no releases with new versions. so maybe it makes sense to specify here both versions - actual released and unreleased, as we do in other modules